std/bytes/mod.ts

```diff
@@ -77,7 +77,7 @@ export function hasPrefix(source: Uint8Array, prefix: Uint8Array): boolean {
  * @param source source array
  * @param suffix suffix array to check in source
  */
```

Comment:
Can you mention the RangeError in the JSDoc?

Reply:
Yes I can do that